query_generators/anthropic.py

```diff
@@ -114,12 +139,34 @@ def generate_query(
             elif columns_to_keep == 0:
                 md = dbs[self.db_name]["table_metadata"]
                 pruned_metadata_str = to_prompt_schema(md, shuffle)
+                pruned_metadata_str = convert_ddl(
+                    postgres_ddl=pruned_metadata_str,
+                    to_dialect=self.db_type,
+                    db_name=self.db_name,
+                )
```

Comment:
meaning we only do translation if there's no pruning? in gen_prompt.py the translation occurs only after the pruning/non-pruning

Reply:
Will fix, thanks for spotting!